app/grandchallenge/discussion_forums/views.py

```diff
@@ -245,3 +268,20 @@ def get_form_kwargs(self):
             }
         )
         return kwargs
+
+
+class MyForumPosts(ViewObjectPermissionListMixin, ListView):
+    model = ForumPost
+
+    def get_queryset(self):
+        qs = super().get_queryset()
+        return qs.filter(creator=self.request.user)
+
+    @cached_property
+    def forum(self):
+        return self.request.challenge.discussion_forum
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data()
+        context.update({"forum": self.forum})
```

Comment:
Does it make sense to show posts from other forums in the context of this forum?